worksheet_A/TerminalHacking.cpp

```diff
@@ -34,13 +57,45 @@ int main()
 	}
 
 	// Display all words
-	for each (std::string word in options)
+    for(std::string word : options)
 	{
 		std::cout << word << std::endl;
 	}
 
 	// TODO: implement the rest of the game
-
+    while(lives>0)//Game loop
+    {
+        std::string guessedWord;        //Define guessedWord outside the Guess loop
+        
+        //Enter Guess. If word is in wordList continue.
+        while(true)
+        {
+            printf("Enter your Guess!(You have %d lives left)\n",lives);
+            std::cin >> guessedWord;    //Input Guess
+            //If the guessed word is in the list return true and break the loop else print Invalid guess
+            if(CheckList(options,guessedWord) == true)
+                break;
+            else
+                printf("Invalid guess\n");
+        }//End Guess loop
+        
+        //Check the guessed word against the secret word
```

Comment:
Good use of comments. However to my eyes the mixture of whole-line and end-of-line comments, and the lack of blank lines before some whole-line comments, makes your code look a little untidy.